power-control-x86/src/power_control.cpp

```diff
@@ -2071,34 +2080,34 @@ static void idButtonHandler()
                              });
 }
 
-static void postCompleteHandler()
-{
-    gpiod::line_event gpioLineEvent = postCompleteLine.event_read();
+// static void postCompleteHandler()
+// {
+//     gpiod::line_event gpioLineEvent = postCompleteLine.event_read();
 
-    bool postComplete =
-        gpioLineEvent.event_type == gpiod::line_event::FALLING_EDGE;
-    if (postComplete)
-    {
-        sendPowerControlEvent(Event::postCompleteAssert);
-        osIface->set_property("OperatingSystemState", std::string("Standby"));
-    }
-    else
-    {
-        sendPowerControlEvent(Event::postCompleteDeAssert);
-        osIface->set_property("OperatingSystemState", std::string("Inactive"));
-    }
-    postCompleteEvent.async_wait(
-        boost::asio::posix::stream_descriptor::wait_read,
-        [](const boost::system::error_code ec) {
-            if (ec)
-            {
-                std::cerr << "POST complete handler error: " << ec.message()
-                          << "\n";
-                return;
-            }
-            postCompleteHandler();
-        });
-}
+//     bool postComplete =
+//         gpioLineEvent.event_type == gpiod::line_event::FALLING_EDGE;
+//     if (postComplete)
+//     {
+//         sendPowerControlEvent(Event::postCompleteAssert);
+//         osIface->set_property("OperatingSystemState", std::string("Standby"));
+//     }
+//     else
+//     {
+//         sendPowerControlEvent(Event::postCompleteDeAssert);
+//         osIface->set_property("OperatingSystemState", std::string("Inactive"));
+//     }
+//     postCompleteEvent.async_wait(
+//         boost::asio::posix::stream_descriptor::wait_read,
+//         [](const boost::system::error_code ec) {
+//             if (ec)
+//             {
+//                 std::cerr << "POST complete handler error: " << ec.message()
+//                           << "\n";
+//                 return;
+//             }
+//             postCompleteHandler();
+//         });
+// }
```

Comment:
can we use the #if 0 & #endif instead?

Reply:
Okay, I will change them from comment out to #if 0 & #endiif